docs/UserGuide.md

```diff
@@ -180,6 +182,41 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Creating an event: `create`
+
+Creates a new event in the address book.
+
+Format: `create t/EVENT_TITLE d/DATE t/TIME p/PURPOSE`
+
+* The compulsory parameters are: event title (t), date (d), time (t) and purpose (p)
+
+Examples:
+* `create t/Shoe Sale 30% d/30-05-2022 t/11:00 p/Discount on all shoes for up to 30%`
+* `create t/Banana Discount 10% d/20-04-2022 t/14:00 p/10% discount on all bananas`
+
+### Removing an event: `remove`
+
+Removes an existing event in the address book.
+
+Format: `remove INDEX`
+
+* Removes the event at the specified INDEX. 
+* The index refers to the index number shown in the displayed person list 
+* The index must be a positive integer 1, 2, 3, …, and it must be within the range of the event list index.
+
+Examples:
+* `delete 2 after displaying all events with display removes event at index 2`
+* `delete 7 after displaying all events with display removes event at index 7`
```

Comment:
I think it's better that only the actual command be formatted as inline code; the description should be in plain text.
```suggestion
* `delete 2` after displaying all events with display removes event at index 2
* `delete 7` after displaying all events with display removes event at index 7
```

Reply:
Yes, thanks! Shouldn't have formatted the whole line